docs/TEMPLATING.md

```diff
@@ -150,22 +245,33 @@ the `.` value passed in. You can test this with `chezmoi cat`:
 	fontsize: 18
 	somemore: config
 
-This approach only works for a single value. If you want to pass in more than one value you can pass in structured data from your config file:
+### Passing multiple arguments
+In the example above only one arguments is passed to the template. To pass
+more arguments to the template, you can do it in two ways.
+
+#### Via chezmoi.toml
+
+This method is useful if you want to use the same template arguments multiple
+times, because you don't specify the arguments every time. Instead you specify
+them in the file `.chezmoi.toml`.
 
-`.config/chezmoi/chezmoi.toml`
+`.local/share/chezmoi/.chezmoi.toml`:
```

Comment:
This should be `.config/local/chezmoi.toml`.

Reply:
Thanks, I've corrected it